analysis/cohensKappa.py

```diff
@@ -0,0 +1,31 @@
+"""
+Takes a csv of the "compare scores" sheet
+Calculated and prints the relaxed and strict Cohen's kappa scores
+You'll need to modify the indices and remove a few lines
+when you're no longer doing two types of scores
+"""
+
+import csv
+from sklearn.metrics import cohen_kappa_score
+
+rater1R = []  # First rater, relaxed
+rater1S = []  # First rater, strict
+rater2R = []  # Second rater, relaxed
+rater2S = []  # Second rater, strict
+
+with open("/Users/Laura/Desktop/NIH/gend-intel-scoring/compare.csv") as f:
+    reader = csv.reader(f)
+    header = next(reader)
+    header = next(reader)  # There are two header lines in the spreadsheet
+    for line in reader:
+        if line[4] != '':  # Skip any sentences that haven't been scored yet
+            rater1R.append(int(line[4]) + int(line[7]) + int(line[10]) +
+                           int(line[13]) + int(line[16]))
+            rater1S.append(int(line[5]) + int(line[8]) + int(line[11]) +
+                           int(line[14]) + int(line[17]))
+            rater2R.append(int(line[19]) + int(line[22]) + int(line[25]) +
+                           int(line[28]) + int(line[31]))
+            rater2S.append(int(line[20]) + int(line[23]) + int(line[26]) +
+                           int(line[29]) + int(line[32]))
```

Comment:
there is something about this script that I really don't like: the user has to take it on faith that the groups column numbers specified here (lines 22-29) are correct.  The only safeguard in place is that (presumably) the `cohen_kappa_score` function will bomb if you pass it strings instead of numbers; other than that it's entirely possible for the columns in the original to get shuffled, and to get the wrong result and not know it.  I'm editing this to show how I would improve that, and will submit a PR into your fork shortly.

Reply:
(in the meantime, leave this PR open)